turn/turn_rest.go

```diff
@@ -0,0 +1,30 @@
+package turn
+
+import (
+	"net"
+
+	"github.com/screego/server/config"
+)
+
+type TurnREST struct {
+}
+
+func (t *TurnREST) Allow(username, password string, addr net.IP) {
+	//FIXME
```

Comment:
Hey, thanks for your contribution!

Will the implementation be done in this PR?

Reply:
I'm struggling with a clean interface, usable with embeded turn service and coturn : turn/turn.go

I'm implementing the authentication with hmac : turn/turn_rest.go